rwengine/src/audio/SoundSource.cpp

```diff
@@ -254,17 +254,14 @@ static int read_packet(void* opaque, uint8_t* buf, int buf_size) {
     return buf_size;
 }
 
-void SoundSource::loadSfx(const rwfs::path& path, LoaderSDT& sdt, size_t index, bool asWave) {
+void SoundSource::loadSfx(LoaderSDT& sdt, size_t index, bool asWave) {
```

Comment:
Can the Loader be const here? Just wondering.

Reply:
Cannot.

Reply:
Why? I need raw_data  to live enough for feeding ffmpeg, it requires life time longer than function's one, so I store it in "body" of sdt.

Reply:
I was refering to this "Little Britain" meme
https://www.youtube.com/watch?v=AJQ3TM-p2QI